site-manage/site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/impl/SiteManageGroupSectionRoleHandler.java

```diff
@@ -1611,14 +1619,10 @@ private boolean processUploadGroupLines(List<String[]> lines){
 				messages.addMessage(new TargettedMessage("import1.error.invalid.data.format", null, TargettedMessage.SEVERITY_ERROR));
 				return false;
 			}
-			// if we already have an occurrence of this group, get the group, check whether the user is already there, if not add them.
+			// if we already have an occurrence of this group add the user, otherwise create a new group.
 			if(groupMap.containsKey(groupTitle)){
 				ImportedGroup group = groupMap.get(groupTitle);
-				for (String s: group.getUserIds()) {
-					if (!s.equals(userId)) {
-						group.addUser(userId);
-					}
-				}
+				group.addUser(userId);
```

Comment:
This change works because the collections of members inside `ImportedGroup` is a set so there's no harm is adding and already present user.

Reply:
so can this be merged then?